tests/test_validators.py

```diff
@@ -683,21 +686,28 @@ def check_a(cls, v: Any):
     assert check_calls == 1
 
 
-def test_wildcard_validators():
+@pytest.mark.parametrize(
```

Comment:
If you don't like this, I can create a separate test - but it will be the same logic.

Reply:
fine for me